src/ECharts/Chart.js

```
@@ -29,17 +29,23 @@ exports.registerTheme = function(name) {
 exports.setOptionImpl = function(option) {
     return function(chart) {
         return function() {
-            chart.setOption(option, false, false);
-            return {};
+            try {
+                chart.setOption(option, false, false);
+            } catch (e) {
+                connsole.error(e);

```

Typo on `console`

---

Also, is this a good idea anyway? What's the thinking by making it a silent error?

---

Well, that's provide at least a message, because halogen for example eats all exceptions and silently breaks